python/mxnet/executor.py

```diff
@@ -26,7 +26,15 @@ def callback_handle(name, array, _):
     return callback_handle
 
 class Executor(object):
-    """Executor is the actual executing object of MXNet."""
+    """Executor is the actual executing object of MXNet,
+       which provides efficient symbolic graph execution
+       and optimization.
+
+    Examples
+    --------
+    >>> # typical approach to create an executor is to bind symbol
+    >>> texec = symbol.bind(ctx=context, args=arg_arrays)
```

Comment:
bind and simple bind. use real mx.cpu(0) instead of context. replace arg_arrays with a real dict